src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -180,6 +180,11 @@
     </div>
 {%- endblock choice_widget_expanded %}
 
+{% block file_widget -%}
```

Comment:
In the `form_simple_widget` there is a test on the type of form, if it's a file, it change the class. Adapt it is a better idea than do this block, no ?

    {%- set attr = attr|merge({class: (attr.class|default('') ~ ' form-control' ~ (type|default('') == 'file' ? '-file' : ''))|trim}) -%}

Dot it:

    {%- set attr = attr|merge({class: (attr.class|default('') ~ (type|default('') == 'file' ? 'custom-file-control' : 'form-control'))|trim}) -%}

Reply:
:+1: